### DISCUSSION TITLE
Translation plugin for Laravel

### DISCUSSION THREAD

**Discussion by ReedDee** [spatie/laravel-translatable#257] [Q&A]
Hello,
Is there a plug or SW that can help me translate my laravel site - something similar to WPML for WP.

I would like to extract all strings in english in order to translate those to French and easily integrate those strings to achieve the expected translation.  

Asking this on behalf of my developer as he is proposing something that is not workable for as you can see in the attached file. 

![image_2021_01_21T13_36_09_255Z](https://user-images.githubusercontent.com/77796823/105378934-3a241580-5c04-11eb-89c8-0b62130542cc.png)

**Comment by boryn** [comment]
Check this library: https://github.com/nikaia/translation-sheet - it exports used strings to Google Sheet where you have a convenience of sharing it with other users and later after having them translated you import them back.
